load-agent/agents/verifier/acapy.py

```diff
@@ -80,96 +82,77 @@ def is_up(self):
+                                                item["name"]: {"name": item["name"]}
+                                                for item in self.cred_attributes
+                                        },
+                                        requested_predicates={},
+                                        version='1.0'
+                                )
+                        ).model_dump(),
+                        headers=self.headers,
                 )
 
                 try:
                         if r.status_code != 200:
                                 raise Exception("Request was not successful: ", r.content)
-                except JSONDecodeError as e:
+                except JSONDecodeError:
                         raise Exception(
-                                "Encountered JSONDecodeError while parsing the request: ", r
+                                "Encountered JSONDecodeError while parsing the request: ", r.text
                         )
                 
-                r = r.json()
-
-                return r
+                return r.json()
         
         def request_verification(self, connection_id):
                 # From verification side
-                headers = json.loads(os.getenv("VERIFIER_HEADERS"))  # headers same
-                headers["Content-Type"] = "application/json"
-
-                verifier_did = os.getenv("CRED_DEF").split(":")[0]
-                schema_parts = os.getenv("SCHEMA").split(":")
-
                 # Might need to change nonce
                 # TO DO: Generalize schema parts
                 r = requests.post(
-                        os.getenv("VERIFIER_URL") + "/present-proof/send-request",
-                        json={
-                                "auto_remove": False,
-                                "auto_verify": True,
-                                "comment": "Performance Verification",
-                                "connection_id": connection_id,
-                                "proof_request": {
-                                "name": "PerfScore",
-                                "requested_attributes": {
-                                        item["name"]: {"name": item["name"]}
-                                        for item in json.loads(os.getenv("CRED_ATTR"))
-                                },
-                                "requested_predicates": {},
-                                "version": "1.0",
-                                },
-                                "trace": True,
-                        },
-                        headers=headers,
+                        f"{self.agent_url}/present-proof/send-request",
+                        json=RequestPresentationV1(
+                                comment='Performance Verification',
+                                connection_id=connection_id,
+                                proof_request=ProofRequest().model_dump(
```

Comment:
You put the kwargs in the model_dump function instead of the ProofRequest class __init__. Causes an exception.